components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -166,21 +156,6 @@ class PageWallet extends React.Component<Props, State> {
     })
   }
 
-  toggleQRCodeModal = () => {
-    // If we are opening the QR code panel, then request a payment ID if we do
-    // not already have one and close the backup/restore modal.
-    if (!this.state.modalQRCode) {
-      if (!this.props.rewardsData.paymentId) {
-        this.actions.getPaymentId()
```

Comment:
It seems that `getPaymentId()` in `rewards_actions.ts` is no longer used, also `types.GET_PAYMENT_ID` and the corresponding handler code in `rewards_reducer.ts`.

Reply:
Removed.